_codux/boards/color-select.board.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState } from 'react';
 import { createBoard } from '@wixc3/react-board';
-import { ColorSelect, ColorSelectOption } from '~/components/color-select/color-select';
+import { ColorSelect, ColorSelectOption } from 'lib/components/color-select/color-select';
```

Comment:
not sure about moving components to the lib.. might be confusing for the user why some components are in `lib` and some in `src`. For example `Accordion` sounds like generic component and it's in the `src`, on the other hand - `RangeSlider`, also looks like a generic component but it's in the `lib`.